packages/server/src/seed/seed.controller.ts

```diff
@@ -37,6 +37,8 @@ export class SeedController {
     await this.seedService.deleteAll(EventModel);
     await this.seedService.deleteAll(UserModel);
     await this.seedService.deleteAll(CourseModel);
+    const manager = getManager();
+    manager.query('ALTER SEQUENCE user_model_id_seq RESTART WITH 1;');
```

Comment:
What does this do?

Reply:
We're deleting all the UserModels above, but when you go back to re-seed data the UserModel ids would not start back off at 1. We have dev urls that depend on those user ids though, so we have to run a query to reset the ids since they don't automatically reset after deleting.